File: study08/backend/next/company/models/Qna.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database');

const Qna = sequelize.define('Qna', {
    qno: { type: DataTypes.INTEGER, autoIncrement: true, primaryKey: true },
    lev: { type: DataTypes.INTEGER, defaultValue: 0 },
    parno: { type: DataTypes.INTEGER },
    title: { type: DataTypes.STRING },
    content: { type: DataTypes.TEXT },
    author: { type: DataTypes.STRING },
    resdate: { type: DataTypes.DATE, defaultValue: DataTypes.NOW },
    hits: { type: DataTypes.INTEGER, defaultValue: 0 }
}, {
    tableName: 'qna',
    timestamps: false
});

module.exports = Qna;

File: study08/backend/springframework5/company/src/main/java/com/company/domain/Qna.java
package com.company.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Qna {
    private int qno;
    private int lev;
    private int parno;
    private String title;
    private String content;
    private String author;
    private String resdate;
    private int hits;
}
